src/host_triple.rs

```diff
@@ -1,15 +1,17 @@
 use crate::emoji;
 use anyhow::{Context, Result};
 use guess_host_triple::guess_host_triple;
+use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 use strum::Display;
 use strum_macros::EnumString;
 
-#[derive(Display, Debug, Clone, EnumString)]
+#[derive(Display, Debug, Clone, EnumString, Deserialize, Serialize, Default)]
 pub enum HostTriple {
     /// 64-bit Linux
     #[strum(serialize = "x86_64-unknown-linux-gnu")]
-    X86_64UnknownLinuxGnu = 0,
+    #[default]
```

Comment:
Just a note that this requires a minimum Rust version of 1.62.0; probably not a bad idea to add `rust-version` to the Cargo manifest. If you would like to keep a lower MSRV you will need to implement `Default` manually.

Reply:
Nice catch! I am happy having MSRV at `1.62` but I really don't know if that it's too recent for some users, I'll add the `rust-version` field for the moment and if someone complains about it I'll change it.